### PR TITLE
"docker network" missing from docker --help

### PR DESCRIPTION
**Pull request by kunalkushwaha**
Signed-off-by: Kunal Kushwaha kunal.kushwaha@gmail.com

### CODE REVIEW

File: docker/flags.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ var (
 		{"login", "Register or log in to a Docker registry server"},
 		{"logout", "Log out from a Docker registry server"},
 		{"logs", "Fetch the logs of a container"},
+		{"network", "Manages docker network"},
```
Comment (mavenugo) [inline docker/flags.go]: network and service are experimental features. these must show up only on experimental builds.

Comment (kunalkushwaha) [inline docker/flags.go]: what is procedure for putting code in experimental ?

Comment (mountkin) [inline docker/flags.go]: I think you can create a "flags_experimental.go" file in the "docker" directory and add the "// +build experimental" build tag at the top of the file.
Then you can add the network command in a "init" function in the new file.
For example:
```diff
diff --git a/docker/flags_experimental.go b/docker/flags_experimental.go
new file mode 100644
index 0000000..3b7f32d
--- /dev/null
+++ b/docker/flags_experimental.go
@@ -0,0 +1,7 @@
+// +build experimental
+
+package main
+
+func init() {
+	dockerCommands = append(dockerCommands, command{"network", "Network management"})
+}
```

**Comment by icecrime** [comment]
@kunalkushwaha Thanks for the contrib! Are you available to do the changes as @mountkin suggest?

**Comment by kunalkushwaha** [comment]
@icecrime Yes, Will create a new PR.  I thought it was not required, so didn't followed up.

**Comment by kunalkushwaha** [comment]
Created a new PR #14626

**Pull request closed without merging**